### PR TITLE
Extracted quotes to google spreadsheet + added Konami code functionality

### PR DESCRIPTION
**Pull request by jesiggar**
Added Konami code functionality

### CODE REVIEW

File: src/rajoyirize.js [hunk 1]
```diff
@@ -61,32 +61,82 @@
...
+        function displayQuote() {
+            properties.DOM.quoteContainer.innerHTML = getQuote();
+            properties.DOM.rajoyirizeContentWrapper.classList.add('active');
+            hideQuote();
+        }
+
+        function hideQuote() {
+            var timer = w.setTimeout(function() {
+                properties.DOM.rajoyirizeContentWrapper.classList.remove('active');
+                clearTimeout(timer);
```
Comment (aPlana) [inline src/rajoyirize.js:101]: Don't think this is necessary, but ¯\\\_(ツ)_/¯

File: src/rajoyirize.js [hunk 2]
```diff
@@ -61,32 +61,82 @@
...
         if (properties.settings.autostart) {
-            this.start();
+            start();
+        }
+
+        // Konami code listener
+        d.onkeydown = function(event) {
+            userKeys.push(event.keyCode)
+
+            if (userKeys.toString().indexOf(konamiKeys) >= 0) {
```
Comment (aPlana) [inline src/rajoyirize.js:133]: This is a bit wtf way to do it, but hey, it works 😂

**Review by aPlana — APPROVED** [review]
👍

File: src/rajoyirize.js [hunk 3]
```diff
@@ -61,32 +61,82 @@
         }
 
         function getQuotes() {
-            return quotes || [{ quote: 'no quotes file found'}];
+            var request = new XMLHttpRequest();
+            var quotesSpreadsheetUrl = 'https://spreadsheets.google.com/feeds/list/1tMu-I25eZX1ojpSnB3W4ww6SB9uAbsSKFEwBLmX81F0/od6/public/values?alt=json';
```
Comment (aPlana) [inline src/rajoyirize.js:65]: Is the spreadsheet public? How dangerous is this?

Comment (jesiggar) [reply src/rajoyirize.js:65]: We have to find a better solution to store the quotes, but thought that was a good option in the meantime. The spreadsheet is public but not editable.

**Pull request merged**